lib/Pod/To/Text.pm6

```diff
@@ -33,14 +33,32 @@ sub pod2text($pod) is export {
 
 sub heading2text($pod) {
     given $pod.level {
-        when 1  {          pod2text($pod.contents)  }
-        when 2  { '  '   ~ pod2text($pod.contents)  }
+        my $content = $pod.contents.first.contents.grep( * !~~ "").first;
+        my $text;
+        if $content ~~ Str {
+            $text = $content;
+        } else {
+            $text = pod2text $content;
+        }
+        my $name = $text.lines[0];
+        my $rest = $text.lines[1…*].join: "\n";
+        when 1  {
+            my $result = colored($name.uc, 'bold') ~ "\n";
+            $result ~= "\n$rest" if $rest.defined;
+            $result.trim;
+        }
+        when 2  {
+            my $result = '  ' ~ colored($name, 'bold') ~ "\n";
+            $result ~= "\n$rest" if $rest.defined;
+            $result.trim-trailing;
+        }
         default { '    ' ~ pod2text($pod.contents)  }
     }
 }
 
 sub code2text($pod) {
-    "    " ~ $pod.contents>>.&pod2text.subst(/\n/, "\n    ", :g)
+    my $text = "    " ~ $pod.contents>>.&pod2text.subst(/\n/, "\n    ", :g);
```

Comment:
This seems to just be indenting the code… We **do** have the `.indent` function.
`$pod.contents>>.&pod2text.indent(4)`